docs/configuration/config-file-reference.md

```diff
@@ -975,6 +975,28 @@ lifecycler:
 # After what time a series is considered to be inactive.
 # CLI flag: -ingester.active-series-metrics-idle-timeout
 [active_series_metrics_idle_timeout: <duration> | default = 10m]
+
+global_limits:
+  # Max ingestion rate (samples/sec) that ingester will accept. This limit is
+  # per-ingester, not per-tenant. Additional push requests will be rejected. 0 =
+  # unlimited.
+  # CLI flag: -ingester.global-limits.max-ingestion-rate
+  [max_ingestion_rate: <float> | default = 0]
+
+  # Max users that this ingester can hold. Requests from additional users will
+  # be rejected. 0 = unlimited.
+  # CLI flag: -ingester.global-limits.max-users
+  [max_users: <int> | default = 0]
```

Comment:
In issue #2987 it seems like there's an agreement to use `tenant` as word instead of using `users`. Does `user` refer to something else than a tenant in this context? If not: Does it make sense to keep going with `tenant` so that a potential rename in the future could be avoided?

Reply:
I agree on this feedback too.

Reply:
> Does `user` refer to something else than a tenant in this context?

No, it's the same. I'll switch to tenants.